test/WebJobs.Script.Tests/Description/DotNet/PackageAssemblyResolverTests.cs

```diff
@@ -43,7 +43,14 @@ public void Dispose()
         {
             if (Directory.Exists(_runPath))
             {
-                Directory.Delete(_runPath, true);
+                try
+                {
+                    Directory.Delete(_runPath, true);
```

Comment:
When running unit tests, getting frequent delete conflicts due to file locking issues